src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/TribunalsHearingsEventQueueListener.java

```diff
@@ -52,7 +52,7 @@ public void handleIncomingMessage(HearingRequest message) throws TribunalsEventP
         HearingState event = message.getHearingState();
 
         log.info("Attempting to process hearing event {} from hearings event queue for case ID {}",
-                event, caseId);
+                 event, caseId);
```

Comment:
is it easy for you to get rid off all the reformatting-only changes.... it just clutters up the PR

Reply:
done